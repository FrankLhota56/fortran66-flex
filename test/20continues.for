C
C Tests more than 19 Continues
C
     0ISUM =  0 +
     1        1 +
     2        2 +
     3        3 +
     4        4 +
     5        5 +
     6        6 +
     7        7 +
     8        8 +
     9        9 +
     X       10 +
     1       11 +
     2       12 +
     3       13 +
     4       14 +
     5       15 +
     6       16 +
     7       17 +
     8       18 +
     9       19 +
     X       20
      END
