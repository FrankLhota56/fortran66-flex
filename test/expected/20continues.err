More than 19 continuation lines
