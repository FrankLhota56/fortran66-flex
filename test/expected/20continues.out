Line     4: ID(isum)
Line     4: '='
Line     4: INT_LIT(0)
Line     4: '+'
Line     5: INT_LIT(1)
Line     5: '+'
Line     6: INT_LIT(2)
Line     6: '+'
Line     7: INT_LIT(3)
Line     7: '+'
Line     8: INT_LIT(4)
Line     8: '+'
Line     9: INT_LIT(5)
Line     9: '+'
Line    10: INT_LIT(6)
Line    10: '+'
Line    11: INT_LIT(7)
Line    11: '+'
Line    12: INT_LIT(8)
Line    12: '+'
Line    13: INT_LIT(9)
Line    13: '+'
Line    14: INT_LIT(10)
Line    14: '+'
Line    15: INT_LIT(11)
Line    15: '+'
Line    16: INT_LIT(12)
Line    16: '+'
Line    17: INT_LIT(13)
Line    17: '+'
Line    18: INT_LIT(14)
Line    18: '+'
Line    19: INT_LIT(15)
Line    19: '+'
Line    20: INT_LIT(16)
Line    20: '+'
Line    21: INT_LIT(17)
Line    21: '+'
Line    22: INT_LIT(18)
Line    22: '+'
Line    23: INT_LIT(19)
Line    23: '+'
